### PR TITLE
[icons] feat: new icons fuel, playbook

### PR DESCRIPTION
**Pull request by pierceashworth98**
#### Checklist

- [ ] Includes tests
- [ ] Update documentation

#### Changes proposed in this pull request:

Added icons for 'fuel' and 'playbook'

#### Reviewers should focus on:

Making sure everything renders correctly and the right files were changed

image

### CODE REVIEW

File: resources/icons/20px/playbook.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<path fill-rule="evenodd" clip-rule="evenodd" d="M16 20C18.2091 20 20 18.2091 20 16C20 13.7909 18.2091 12 16 12C13.7909 12 12 13.7909 12 16C12 18.2091 13.7909 20 16 20ZM16 18C17.1046 18 18 17.1046 18 16C18 14.8954 17.1046 14 16 14C14.8954 14 14 14.8954 14 16C14 17.1046 14.8954 18 16 18Z" fill="black"/>
+<path d="M18.29 5.7L16 3.41V8C16 9.65685 14.6569 11 13 11H4C3.44772 11 3 11.4477 3 12V19C3 19.5523 2.55228 20 2 20C1.44772 20 1 19.5523 1 19V12C1 10.3431 2.34315 9 4 9H13C13.5523 9 14 8.55228 14 8V3.41L11.71 5.71C11.53 5.89 11.28 6 11 6C10.45 6 10 5.55 10 5C10 4.72 10.11 4.47 10.29 4.29L14.29 0.29C14.47 0.11 14.72 1.49012e-08 15 1.49012e-08C15.28 1.49012e-08 15.53 0.11 15.71 0.29L19.71 4.29C19.89 4.47 20 4.72 20 5C20 5.55 19.55 6 19 6C18.72 6 18.47 5.89 18.29 5.7Z" fill="black"/>
+</g>
+<defs>
+<clipPath id="clip0_328_94">
+<rect width="20" height="20" fill="white"/>
+</clipPath>
+</defs>
```
Comment (adidahiya) [inline resources/icons/20px/playbook.svg]: I think this `<defs>` is extraneous

```suggestion
</g>
```

File: packages/icons/icons.json [hunk 2]
```diff
@@ -3820,5 +3820,17 @@
         "tags": "symbol, rectangle, filled shape",
         "group": "interface",
         "codepoint": 62242
+    },
+    {
+        "displayName": "Fuel",
+        "iconName": "fuel",
+        "tags": "fuel, gas, gas can, refuel",
+        "group": "interface"
```
Comment (adidahiya) [inline packages/icons/icons.json]: I updated the icons contribution guide to mention the new codepoint information required... you'll need to add these:

```suggestion
        "group": "interface",
        "codepoint": 62243
```

File: packages/icons/icons.json [hunk 3]
```diff
@@ -3820,5 +3820,17 @@
...
+    },
+    {
+        "displayName": "Playbook",
+        "iconName": "playbook",
+        "tags": "strategy, play, plan, course of action",
+        "group": "interface"
```
Comment (adidahiya) [inline packages/icons/icons.json]:
```suggestion
        "group": "interface",
        "codepoint": 62244
```

**Review by adidahiya — CHANGES_REQUESTED** [review]
Note to self: the current error message for undefined codepoints is not great, I will fix this to make it more legible:

```
Error: [generate-icon-fonts] Invalid metadata entry in icons.json: icon "playbook" cannot have codepoint undefined, it is already in use.
```

**Comment by pierceashworth98** [comment]
@adidahiya I updated it according to your suggestions. Hopefully it should be good now, if not I'll ask one of my devs to help.

**Comment by adidahiya** [comment]
looks good in [docs preview](https://52658-71939872-gh.circle-artifacts.com/0/packages/docs-app/dist/index.html#icons) 👍🏽 

image
image

**Review by adidahiya — APPROVED** [review]

**Pull request merged**